component/azstorage/azstorage.go

```diff
@@ -387,7 +387,7 @@ func (az *AzStorage) TruncateFile(options internal.TruncateFileOptions) error {
 		}
 	}
 
-	err = az.storage.WriteFromBuffer(options.Name, nil, data)
+	err = az.storage.WriteFromBuffer(options.Name, attr.Metadata, data)
```

Comment:
If attr-cache is not in the pipeline, it shall come down as nil. We may need to make changes in libfuse layer for this.

Reply:
Is there a problem with it being nil? We were passing nil before. The GO SDK can handle it

Reply:
No what I mean here is shall we explicitly pass nil where this call originates like libfuse or in file-cache or we want to rely on the default value being nil

Reply:
Oh I see, we can pass nil thats fine